tests/rt.sh

```diff
@@ -632,15 +632,15 @@ EOF
 
     # Set RT_SUFFIX (regression test run directories and log files) and BL_SUFFIX
     # (regression test baseline directories) for REPRO or PROD runs
-    if [[ ${MAKE_OPT^^} =~ "REPRO=Y" ]]; then
+    if [[ ${MAKE_OPT^^} =~ "-DREPRO=ON" ]]; then
       RT_SUFFIX="_repro"
       BL_SUFFIX="_repro"
     else
       RT_SUFFIX=""
       BL_SUFFIX=""
     fi
 
-    if [[ ${MAKE_OPT^^} =~ "WW3=Y" ]]; then
+    if [[ ${MAKE_OPT^^} =~ "-DAPP=ATMW" ]] ||  [[ ${MAKE_OPT^^} =~ "-DAPP=S2SW" ]]; then
```

Comment:
I think that this is not flexible that I explain in the following issue https://github.com/ufs-community/ufs-weather-model/issues/656. I already completed WW3 coupling through the use of CMEPS under HAFS application and created a set of wave coupled configurations. Listing applications names in this level is not flexible and once someone bring another configuration this file needs to be changed.

Reply:
Please make a suggestion/PR and we can review.
All the applications that are present in the UFS-weather-model are tested and supported.  So if someone brings in an "application", that needs to be supported.
The entire business of adding `if-then-else` needs to be revisited.